static/app/components/forms/projectMapperField.tsx

```diff
@@ -279,15 +305,13 @@ export class RenderField extends Component<RenderProps, State> {
           <NextButtonPanelAlert type="muted">
             <NextButtonWrapper>
               {nextDescription ?? ''}
-              <Button
-                type="button"
-                size="small"
-                priority="primary"
-                icon={<IconOpen size="xs" color="white" />}
-                href={nextUrl}
-              >
-                {nextButtonText}
-              </Button>
+              {disabledTooltipText ? (
+                <Tooltip title={disabledTooltipText} disabled={!isNextButtonDisabled}>
+                  {<NextButton href={nextUrl} />}
```

Comment:
@leeandher theres no prop for a tooltip title? I don’t think you actually need to wrap the button component like this.

Reply:
Didn't know there was a props for this, thanks!